### DISCUSSION TITLE
Option "Stick the cursor to the content" for task list stop cursor after the checkbox

### DISCUSSION THREAD

**Discussion by pitibalrog** [vslinko/obsidian-outliner#342] [Ideas]
"Stick the cursor to the content" option work great but for tasks list it'll be more usable to stop the cursor at the end of `* [ ]` and not at the end of `* `

It can be the same option or a dedicated one.

**Comment by sinewave** [comment]
+100! 

I use tasks a lot and this is a constant source of friction - please consider adding this 🙏🙏🙏

Has anybody found a workaround?
